unit-tests/src/test/java/org/eclipse/collections/impl/list/mutable/FastListTest.java

```diff
@@ -126,6 +128,16 @@ public void wrapCopy()
         Assert.assertEquals(expected, actual);
     }
 
+    @Test
+    public void fastListProperSuperSetOfArrayList()
```

Comment:
Duplicated code in the 3 classes: please introduce a simple static method that takes two classes as parameter.